github-runner-manager/src/github_runner_manager/reactive/process_manager.py

```diff
@@ -4,37 +4,57 @@
 """Module for managing processes which spawn runners reactively."""
 import logging
 import os
-import shutil
 import signal
 
 # All commands run by subprocess are secure.
 import subprocess  # nosec
+import sys
 from pathlib import Path
 
-from github_runner_manager import constants
 from github_runner_manager.configuration import UserInfo
 from github_runner_manager.reactive.types_ import ReactiveProcessConfig
-from github_runner_manager.utilities import secure_run_subprocess
+from github_runner_manager.utilities import get_reactive_log_dir, secure_run_subprocess
 
 logger = logging.getLogger(__name__)
 
-REACTIVE_RUNNER_LOG_DIR = Path("/var/log/reactive_runner")
-
-PYTHON_BIN = "/usr/bin/python3"
 REACTIVE_RUNNER_SCRIPT_MODULE = "github_runner_manager.reactive.runner"
-REACTIVE_RUNNER_CMD_LINE_PREFIX = f"{PYTHON_BIN} -m {REACTIVE_RUNNER_SCRIPT_MODULE}"
-PID_CMD_COLUMN_WIDTH = len(REACTIVE_RUNNER_CMD_LINE_PREFIX)
-PIDS_COMMAND_LINE = [
-    "ps",
-    "axo",
-    f"cmd:{PID_CMD_COLUMN_WIDTH},pid",
-    "--no-headers",
-    "--sort=-start_time",
-]
 UBUNTU_USER = "ubuntu"
 RUNNER_CONFIG_ENV_VAR = "RUNNER_CONFIG"
 
 
+def _get_python_bin() -> str:
+    """Get the Python interpreter to use for reactive processes.
+
+    Returns the current Python interpreter (sys.executable) to ensure
+    reactive subprocesses use the same Python environment as the parent.
+
+    Returns:
+        The path to the Python interpreter.
+    """
+    return sys.executable
+
+
+# Update these dynamically based on the Python binary
```

Comment:
What is this comment referring to?

Would it be better if this comment is in the docstring?